content-warehouse/src/main/java/contentwarehouse/v1/QuickStart.java

```diff
@@ -0,0 +1,142 @@
+import com.google.cloud.resourcemanager.ProjectInfo;
+import com.google.cloud.resourcemanager.ResourceManager;
+import com.google.cloud.resourcemanager.ResourceManagerOptions;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class QuickStart {
+  
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String userId = "your-user-id"; // Format is user:<user-id>
+    quickStart(projectId, location, userId);
+  }
+
+  public static void quickStart(String projectId, String location, String userId)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = String.format("%s-contentwarehouse.googleapis.com:443", location);
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+         DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+  
+    // Create a Schema Service client
+    try (DocumentSchemaServiceClient documentSchemaServiceClient =
+        DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+      /*  The full resource name of the location, e.g.:
+      projects/{project_number}/locations/{location} */
+      String parent = LocationName.format(projectNumber, location);
+
+      /* Create Document Schema with Text Type Property Definition
+       * More detail on managing Document Schemas: 
+       * https://cloud.google.com/document-warehouse/docs/manage-document-schemas */
+      DocumentSchema documentSchema = DocumentSchema.newBuilder()
+          .setDisplayName("My Test Schema")
+          .setDescription("My Test Schema's Description")
+          .addPropertyDefinitions(
+            PropertyDefinition.newBuilder()
+            .setName("stock_symbol")
+            .setDisplayName("Searchable text")
+            .setIsSearchable(true)
+            .setTextTypeOptions(TextTypeOptions.newBuilder().build())
+            .build()).build();
+
+      // Define Document Schema request
+      CreateDocumentSchemaRequest createDocumentSchemaRequest =
+          CreateDocumentSchemaRequest.newBuilder()
+            .setParent(parent)
+            .setDocumentSchema(documentSchema).build();
+
+      // Create Document Schema
+      DocumentSchema documentSchemaResponse =
+          documentSchemaServiceClient.createDocumentSchema(createDocumentSchemaRequest); 
+        
+
+      // Create Document Service Client Settings
+      DocumentServiceSettings documentServiceSettings = 
+          DocumentServiceSettings.newBuilder().setEndpoint(endpoint).build();
+      TextArray textArray = TextArray.newBuilder().addValues("GOOG").build();
```

Comment:
Might be nice to move this into the try section so it's closer to where it's used. Is there a specific reason for this value?

Reply:
I moved this the textarray section within the try block. I couldn't move settings since it's referenced within the try